src/integration/test.yaml

```diff
@@ -0,0 +1,16 @@
+
+required_packages:
```

Comment:
New complication. Is it even possible for ROS to parse a yaml with an arbitrary sized list of parameters?

Reply:
Would it be too scuffed if the ros_parameter is just the name of a custom yaml config file for the test?

Reply:
An alternative to an arbitrary sized list is a flag for each package

Reply:
I implemented dynamic typing based on the config file fields so there's no way using ROS parameters for the test config file is gonna work.